frontend/src/lib/components/Table.tsx

```diff
@@ -7,6 +7,7 @@ import { TZLabel } from './TimezoneAware'
 export function createdAtColumn(): Record<string, any> {
     return {
         title: 'Created',
+        ellipsis: true,
```

Comment:
Instead of an ellipsis, can't we just "nowrap" this column? The "2 days ago" texts are usually very small, and just having a random "..." in the end feels weird:

![image](https://user-images.githubusercontent.com/53387/122371736-60a26e80-cf60-11eb-8e9c-c8796e2e1528.png)

Reply:
agreed, good shout!

Reply:
Something like this?

<img width="624" alt="Screen Shot 2021-06-21 at 9 44 18 AM" src="https://user-images.githubusercontent.com/13460330/122798294-42e14c00-d275-11eb-9a12-1d4572d01e04.png">

Reply:
Hmm, not really. I meant to just expand the column to fit everything... 🙃

Reply:
Ah how does this look

<img width="513" alt="Screen Shot 2021-06-21 at 10 40 53 AM" src="https://user-images.githubusercontent.com/13460330/122804810-306b1080-d27d-11eb-9557-9278be872517.png">